src/main/java/frc/robot/RobotContainer.java

```diff
@@ -125,48 +125,52 @@ public RobotContainer() {
     drivetrain.zeroyawnavx();
     clawSubsystem.zeroclaw();
     controls.setup();
+    onebar.resetEnc();
   }
 
   private void configureButtonBindings() {
```

Comment:
There are a lot proposed button binding changes here. Did you intend on just commenting out the wrist buttons?  There's a change already in progress that is updating the button bindings to what the drive team requested.
https://github.com/OysterRiverOverdrive/Charged-Up-2023-Atlas/pull/37/files

Reply:
Chatted offline.  The changes are intentional and the requested mapping of buttons to actions have changed.